changelogs/unreleased/gh_10155_make_iproto_resistant_to_misusage.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/iproto
+
+* Fixed an issue when Tarantool could easily crash if iproto was used
+  incorrectly, not according to the protocol (gh-10155).
```

Comment:
- We never used "iproto" section. Better use "core" for this bug.
- I'd be more specific about what happens here.

```suggestion
## bugfix/core

* Fixed a bug when a server could crash if a client sent an IPROTO replication
  request without waiting for pending requests to complete (gh-10155).
```

Reply:
Fixed.